package.json

```diff
@@ -6,6 +6,7 @@
   },
   "dependencies": {
     "react-google-recaptcha": "^2.1.0",
+    "react-router-hash-link": "^2.4.3",
```

Comment:
this is installed in the wrong directory. should be in /ui. right now it's in our project root